cmd/manager/main.go

```diff
@@ -108,7 +114,12 @@ func run() error {
 	}
 
 	log.V(2).Info("Expose metrics")
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+	// TODO: Use operator-custom-metrics
+	servicePorts := []v1.ServicePort{
+		{Port: metricsPort, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metricsPort}},
+	}
+
+	_, err = metrics.CreateMetricsService(ctx, cfg, servicePorts)
```

Comment:
@fahlmant Should we even bother with this, or just blow this whole section away for now?

Reply:
This is the operator-sdk metrics package, which I don't have experience with